platform/posix/transport/src/plaintext_posix.c

```diff
@@ -170,7 +200,29 @@ int32_t Plaintext_Send( const NetworkContext_t * pNetworkContext,
         sendTimeout.tv_usec = 0;
     }
 
+    /* MISRA Directive 4.6 flags the following line for a violation of using a
+     * basic type "int" rather than a type that includes size and signedness information.
+     * We suppress the violation as the flagged type, "fd_set", is a POSIX
+     * system-specific type, and is used for the call to "select()". */
+    /* coverity[misra_c_2012_directive_4_6_violation] */
+    /* coverity[misra_c_2012_rule_14_4_violation] */
```

Comment:
please add the explanation about why 14.4 is marked as ignored here as well

Reply:
Thanks for catching. Added.